perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -761,8 +759,8 @@ sub stash_category_groups : Private {
             $a->[0] cmp $b->[0];
         } @list;
         @list = map { $_->[1] } @list;
-        my $nh = FixMyStreet::Cobrand::HighwaysEngland->new;
-        $nh->national_highways_cleaning_groups(\@list, $c);
+
+        $c->cobrand->call_hook(munge_mixed_category_groups => \@list);
```

Comment:
We don't want every instance of FixMyStreet to be running this code. Here I have replaced cobrand-specific code in core code with a cobrand hook.